python-sdk/src/astro/sql/operators/export_file.py

```diff
@@ -62,6 +73,71 @@ def execute(self, context: Context) -> File:
         else:
             raise FileExistsError(f"{self.output_file.path} file already exists.")
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Collect the input, output, job and run facets for export file operator
+        """
+        input_dataset: list[OpenlineageDataset] = []
+        if isinstance(self.input_data, BaseTable):
```

Comment:
FYI: We emit temp table event based on config https://github.com/astronomer/astro-sdk/pull/1156 you may want to do the same here

Reply:
cc @utkarsharma2 what do you think?